packages/backend/migrations/20230209095515_user.js

```diff
@@ -9,6 +9,8 @@ exports.up = function (knex) {
     table.string('password').notNullable()
     table.string('lastName')
     table.string('firstName')
+    table.string('address')
+    table.string('phoneNumber')
```

Comment:
Where will these two come from, Since they are on the user entity?

Will another from be filled in the frontend when creating a wallet, and the information updated on the user at that time?

Reply:
Yes, exactly. When creating a wallet, address field will be updated.